cookbook/fgcmFitCycleHscCookbook_cycle00_config.py

```diff
@@ -37,8 +37,12 @@
 config.expGrayHighCut = (0.2, 0.2, 0.2, 0.2, 0.2)
 # Number of bins to do aperture correction.  Not currently supported in LSST stack.
 config.aperCorrFitNBins = 0
-# "Fudge factors" for computing SED slope (best values for HSC not determined yet)
-config.sedFudgeFactors = (1.0, 1.0, 1.0, 1.0, 1.0)
+# Aperture correction input slope parameters.  There should be one slope ber band.
+# This is used when there is insufficient data to fit the parameters from the data
+# itself (e.g. tract mode or RC2).
+config.aperCorrInputParameters = (-1.0150, -0.9694, -1.7229, -1.4549, -1.1998)
```

Comment:
A few things on this!  First: "must" is correct, I will change that.  The number of elements is checked by the `fgcm` code, I'll make that clear.  (This is also the case with the other config vars which are per band).
Second: the controlling variable on whether or not to perform a fit is `aperCorrFitNBins` but this is not actually described in the `fgcmcal` documentation.  
So the idea is that these are the input parameters (optional!) and it will use these at the start of the run, and if `aperCorrFitNBins == 0` then the output parameters will equal the input parameters.  Otherwise they will be different.  
This is not explicitly spelled out in the documentation, and I will improve that.  But hopefully this logic does make sense, or do I need another config parameter or to rename something as well?

Reply:
It's convoluted, but as long as it is well-documented, I'm ok with whatever you feel is clear-enough.